### PR TITLE
Mounts storage for /tmp in the container

### PR DESCRIPTION
**Pull request by saltiyazan**
# Description

Mounts storage for /tmp in the container

# Checklist:

- [ ] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that validate the behaviour of the software
- [ ] I validated that new and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: metadata.yaml [hunk 1]
```diff
@@ -45,6 +47,9 @@ storage:
   certs:
     type: filesystem
     minimum-size: 50M
+  tmp:
+    type: filesystem
+    minimum-size: 50M
```
Comment (gruyaume) [inline metadata.yaml:52]: Is 50M enough for backups?

File: metadata.yaml [hunk 2]
```diff
@@ -45,6 +47,9 @@ storage:
   certs:
     type: filesystem
     minimum-size: 50M
```
Comment (Gmerold) [inline metadata.yaml:49]: And isn't 50M for certs "a bit" too much?

Comment (gruyaume) [reply metadata.yaml:49]: True, 1M would be more than enough.

Comment (saltiyazan) [reply metadata.yaml]: I modified the sizes, in the case of tmp it's really hard to estimate, we can't know how big a vault backend can get in a production deployment.
If you have other suggestions let me know.
I modified the sizes of the other volumes too, let me know if I should remove it from this pr.

**Review by gruyaume — APPROVED** [review]

**Review by Gmerold — APPROVED** [review]

**Pull request merged**